plugins/cliconf/ios.py

```diff
@@ -198,6 +198,16 @@ def edit_config(
 
                 cmd = line["command"]
```

Comment:
I believe an alternate and cleaner fix would be to update the terminal error regex.